### ISSUE TITLE
Throws std::bad_cast before crash upon import in python 3.9/3.10

### ISSUE THREAD

**Issue by sadernalwis** [panda3d/panda3d#1361]
## Description
Throws then crashes python environment:
`terminate called after throwing an instance of 'std::bad_cast'`
`what():  std::bad_cast`

## Steps to Reproduce
`from panda3d.core import *`
`from panda3d.core import LVector3f`
## Environment
 * Operating system: Ubuntu 20.04 Focal Fossa
 * System architecture: Intel x64
 * Panda3D version: Unable to get because crash at import. 
 * Installation method: pip used wheel from latest `pip install panda3d`
 * Python version (if using Python): 3.10
 * Compiler (if using C++): None

update:

Collected this from a second installation.
`
Collecting panda3d
  Downloading panda3d-1.10.11-cp39-cp39-manylinux1_x86_64.whl (53.4 MB)
     ━━━╺━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.3/53.4 MB 1.1 MB/s eta 0:00:45`

**Comment by ksmit799** [comment]
Works fine for me on Windows 10 using Python 3.10. Using the latest version of Panda from pip.

**Comment by rdb** [comment]
So, I tried to reproduce it with this, but it all works for me:
```bash 
docker run --rm=true -e DEBIAN_FRONTEND=noninteractive -i ubuntu:focal bash <<EOF
apt -y update
apt -y install python3.9 python3-pip
python3.9 -m pip install panda3d
python3.8 -m pip install panda3d
python3.9 -c 'import panda3d.core'
python3.8 -c 'import panda3d.core'
EOF
```
I will need more information to determine what's different about your setup.

You say you are using Python 3.10, but Ubuntu Focal only ships with 3.8 and 3.9 in the repositories.  How did you obtain Python 3.10?

**Comment by sadernalwis** [comment]
@rdb Im using Blender 3Ds shipped 3.9 and 3.10 with development headers. Anyways I shifted to NumPy solution.
I could provide info if u want to further investigate. Otherwise we can close .
